ITEM.tsv

```diff
@@ -22,11 +22,11 @@ ITEM_20150317_000021	$Recommended for those who desire a heavier and stronger sw
 ITEM_20150317_000022	$Sabre 
 ITEM_20150317_000023	$Primary weapon of the Cavalry, optimized for slashing. The guard on the hilt helps protect the user's hand. 
 ITEM_20150317_000024	$Shamshir 
-ITEM_20150317_000025	$The large curved crescent edge allows for a powerful slash. 
+ITEM_20150317_000025	$A blade that has a long, curved crescent. It dishes out heavy slashes.
 ITEM_20150317_000026	$Scimitar 
-ITEM_20150317_000027	$Its heavy weight is compensated by its very powerful slash. 
+ITEM_20150317_000027	$Its heavy weight is compensated by its very powerful slashes. 
 ITEM_20150317_000028	$Kris 
-ITEM_20150317_000029	$The twisted edge amplifies the damage it inflicts. A sword that is both practical and decorative. 
+ITEM_20150317_000029	$A snakelike sword that is both practical and decorative. 
```

Comment:
I get your point. You see the word amplify as more of a numerical verb. As in you take "damage" as a quantifiable measurement.

What the description is trying to say is that the winding edge inflicts a more significant wound/injury than a straight edge (so the sword makes a more serious looking injury, not necessarily anymore damaging if you quantify it).

Thus:
>Its snakelike edge deepens the wounds it creates, the sword is both practical and decorative.